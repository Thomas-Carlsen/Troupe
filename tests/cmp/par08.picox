let fun alice() =
      receive [
          hn _   => someUndeclaredVar
      ]
in send (spawn alice, 42)
end
