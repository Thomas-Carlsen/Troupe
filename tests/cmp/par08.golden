troupec: bad base function: someUndeclaredVar
